### ISSUE TITLE
"The TorchScript C++ version and the Python version (8.0.213) have a difference of 0.1-0.2 points in confidence."

### ISSUE THREAD

**Issue by datalin-dcing** [ultralytics/ultralytics#15906]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

When using Ultralytics, why is there a difference of 0.1-0.2 points in confidence and a difference of 1-2 pixels in coordinates between the Python version (8.0.213) and the official TorchScript C++ version? What could be the possible reasons for this?

**Comment by Y-T-G** [comment]
Are you running torchscript in Python too, or the PyTorch model?

**Comment by datalin-dcing** [comment]
> Are you running torchscript in Python too, or the PyTorch model?
yes,c++(libtorch) and python(pytorch) using the same model(torchscript)  ,

**Comment by Y-T-G** [comment]
Maybe the output is rounded.
